docs/reference/pinnings.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Pinnings
+---
+
+# Global pinnings & configuration
+
+For consistent builds and inter-artifact compatibility, many of the libraries and projects used to build packages on conda-forge are pinned or locked to a globally controlled value. We call this "pinnings". This includes compilers, runtime libraries, base systems, and others.
+
+:::info
+
+This information is collected here for your convenience, but it might not be up-to-date with the authoritative source: `conda-forge-pinnings`.
+[Check this file for up-to-date values](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml).
+:::
+
+## Base systems
```

Comment:
this would greatly benefit of using tabs :smile: (the tabs'guy ksksks)

Reply:
I don't know how to do that in Docusaurus (yet) and for now I want to keep it simple, but feel free to create an issue!

Reply:
On second thought, tabs are not directly linkable via `#anchors`, which might hinder navigation. I think tabs are ok for examples and so on, but not for reference info maybe?